overlay.py

```diff
@@ -51,10 +51,14 @@ def __init__(self, bike, width=1280, height=740, bg: str = None):
         self.client.on_disconnect = self.on_disconnect
         self.client.on_log = self.on_log
 
+        # Set the video feed status to offline if connection breaks
+        video_topic = f"{str(DAShboard.status_video_feed)}/{self.device}"
+        self.client.will_set(video_topic, dumps({"online": False}), 1, True)
+
         # Set the camera status to offline if connection breaks
-        is_online_topic = f"{str(DAShboard.status_video_feed)}/{self.device}"
+        camera_topic = f"/v3/status/camera/{self.device}"
```

Comment:
This will get changed to use `DAShboard` after [common#9](https://github.com/monash-human-power/common/pull/9) gets merged.